src/overset/OversetManagerSTK.C

```diff
@@ -1381,7 +1386,10 @@ OversetManagerSTK::complete_search(
       if ( infoObject->bestX_ > maxTol || !(bulkData_->is_valid(elem)) ) {
         NaluEnv::self().naluOutputP0() << "Sorry, constraint node for node " << bulkData_->identifier(infoObject->constraintNode_)
             << " does not have an ideal bestX; consider clipping "
-            << infoObject->bestX_ << std::endl;
+            << infoObject->bestX_ << " " << infoObject->nodalCoords_[0] << " "
```

Comment:
Debug output...

Reply:
This was extremely useful for debugging and it might be good to have something like it in the future for failed overset assembly. It is removed for now.